portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,16 +47,27 @@
 public class DataServlet extends HttpServlet {
 DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-  @Override
+    public class Message {
```

Comment:
Put the new class in it's own file Message.java

Reply:
Would this new class file & future ones go in the same directory as the servlets?

Reply:
Yeah same directory and Java package